Yoast/javascript
# Add more English transition words

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Adds the following words to the list that's used for the English transition word assessment: 'note that', 'not only', 'initially', 'as opposed to'.
* [Yoast SEO Free] Adds the following words to the list that's used for the English transition word assessment: 'note that', 'not only', 'initially', 'as opposed to'.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Use 'note that', 'not only', 'initially', or 'as opposed to' in a sentence and see that it is counted as a sentence with a transition word.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes #
